agent/pkg/crane/crane_test.go

```diff
@@ -63,7 +63,8 @@ func TestMain(m *testing.M) {
 
 	// following is executed after every tests are run in this file
 
-	err := k8s.DeleteNamespace(TestNamespace)
+	var tempConfig config.Configuration
+	err := k8s.DeleteNamespace(TestNamespace, &tempConfig)
```

Comment:
inline instantiation instead? &tempConfig{}

Reply:
in other cases too where there is no reference later.

Reply:
Agree!